lib/utils/desyncedAgentCall.ts

```diff
@@ -0,0 +1,23 @@
+import env from '~/env.mjs'
+
+export default async function desyncedAgentCall({
+	route,
+	body
+}: {
+	route: string
+	body: any
+}) {
+	fetch(`${env.NEXTAUTH_URL}/${route}`, {
+		method: 'POST',
+		headers: {
+			'Content-Type': 'application/json',
+			Authorization: `Bearer ${env.NEXTAUTH_SECRET}`
```

Comment:
Is this crazy

Reply:
feels wrong. Should we use a dif env var? Probably
Is there a better way to do our-server to our-server calls securely??

Reply:
Passing a static token over the air feels fkd.. Maybe we can create some ephemeral token that only works for 1 second encrypted against an env var??? Just brainstorming

Reply:
From our chat this morning, seems like ephemeral token (eg. `run.id` or `run.token`) or new env var is the way to go

Reply:
added a new env var "MAIGE_SERVER_TOKEN"